app/assets/stylesheets/design_system_vibereport.scss

```diff
@@ -109,6 +109,21 @@ input, textarea{
   &:focus{
     border:             3px solid $royal-blue ;
   }
+
+  &::-webkit-scrollbar {
+    width: 15px;
+  }
+
+  &::-webkit-scrollbar-track {
+    background-color: $light-grey;
+    border-radius: 15px;
+  }
+
+  &::-webkit-scrollbar-thumb {
+    background-color: $gray-300;
+    box-shadow: 0 3px 4px rgba(0, 0, 0, 0.3);
+    border-radius: 15px;
+  }
```

Comment:
@Maryna-Harasko I think you can add this more code for a nice hover animation
  &::-webkit-scrollbar-thumb:hover {
    background: shade-color($gray-300,10%);
  }